src/complex/Utilities/DataGroupUtilities.cpp

```diff
@@ -153,4 +146,27 @@ std::vector<std::shared_ptr<IDataArray>> GenerateDataArrayList(const DataStructu
   return arrays;
 }
 
+std::vector<DataPath> GetAllChildDataPaths(const DataStructure& dataStructure, const DataPath& parentGroup, DataObject::Type dataObjectType)
```

Comment:
What did we decide for this function? Was it going to return an optional? Or just rethrow?

Reply:
Changed to std::optional. Worked better in the filters and is cleaner code.